2021 01 19 shaun liu pim (#11399)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/unifiedrolemanagementpolicyassignment.md

---
title: "unifiedRoleManagementPolicyAssignment resource type"
description: "A unifiedRoleManagementPolicyAssignment assigns the policy to a specific scope and role definition."
author: "shauliu"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

> # unifiedRoleManagementPolicyAssignment resource type

Reviewer: Add: [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> # unifiedRoleManagementPolicyAssignment resource type

Reviewer: Add: Namespace: microsoft.graph

A unifiedRoleManagementPolicyAssignment assigns the policy to a specific scope and role definition.

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[List unifiedRoleManagementPolicyAssignments](../api/unifiedrolemanagementpolicyassignment-list.md)|[unifiedRoleManagementPolicyAssignment](../resources/unifiedrolemanagementpolicyassignment.md) collection|Get a list of the [unifiedRoleManagementPolicyAssignment](../resources/unifiedrolemanagementpolicyassignment.md) objects and their properties.|
|[Get unifiedRoleManagementPolicyAssignment](../api/unifiedrolemanagementpolicyassignment-get.md)|[unifiedRoleManagementPolicyAssignment](../resources/unifiedrolemanagementpolicyassignment.md)|Read the properties and relationships of an [unifiedRoleManagementPolicyAssignment](../resources/unifiedrolemanagementpolicyassignment.md) object.|

## Properties
|Property|Type|Description|
|:---|:---|:---|
|id|String|Unique identifier for the policy assignment.|
|policyId|String|The id of the policy.|
|roleDefinitionId|String|The id of the role definition where the policy applies. If not specified, the policy applies to all roles.|

> |scopeId|String|The id of the scope where the policy is assigned. E.g. "/", groupId, etc.|

Reviewer: ```suggestion
|scopeId|String|The id of the scope where the policy is assigned. For example, "/" or groupId.|
```

> |scopeType|String|The type of the scope where the policy is assigned. One of Directory, DirectoryRole, Group.|

Reviewer: ```suggestion
|scopeType|String|The type of the scope where the policy is assigned. Possible values: Directory, DirectoryRole, and Group.|
```
